app/controllers/discourse_ai/admin/ai_personas_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module DiscourseAi
+  module Admin
+    class AiPersonasController < ::Admin::AdminController
+      before_action :find_ai_persona, only: %i[show update destroy]
+
+      def index
+        ai_personas =
+          AiPersona.ordered.map { |persona| AiPersonaSerializer.new(persona, root: false) }
+        commands =
+          DiscourseAi::AiBot::Personas::Persona.all_available_commands.map do |command|
+            { id: command.to_s.split("::").last, name: command.name }
+          end
+        render json: { ai_personas: ai_personas, meta: { commands: commands } }
+      end
+
+      def show
+        render json: { ai_persona: @ai_persona }
+      end
+
+      def create
+        ai_persona = AiPersona.new(ai_persona_params)
+        if ai_persona.save
+          render json: { ai_persona: ai_persona }, status: :created
+        else
+          render json: ai_persona.errors, status: :unprocessable_entity
```

Comment:
For all the error ones, you can use `render_json_error(ai_persona)` and it uses `422` by default